nbgrader/apps/baseapp.py

```diff
@@ -88,13 +171,99 @@ class BaseNbConvertApp(BaseNbGraderApp, NbConvertApp):
     aliases = Dict(nbconvert_aliases)
     flags = Dict(nbconvert_flags)
 
+    use_output_suffix = Bool(False)
+    postprocessor_class = DottedOrNone('')
```

Comment:
Yeah, I think you do need to import it yourself, but not if you use `Type`. Though in this case, it doesn't matter, because there's nothing to import.